source/chapters/library.rst

```diff
@@ -347,7 +347,7 @@ a crate, or even the whole library) for tracks that match your search query.
 * To clear the search string hit :kbd:`ESC` or click the clear button right next
   to the input field.
 * Hit :kbd:`TAB` to cycle between the search and the list of results in the
-  library. Use the :kbd:`ARROW UP` and :kbd:`ARROW DOWN` keys to scroll in the
+  library. Use the :kbd:`↑` and :kbd:`↓` keys to scroll in the
```

Comment:
...and let's use UP and DOWN here.

Reply:
I thought we just said we use the regular arrows instead of the more obscure ones.